src/transformers/models/decision_transformer/modeling_decision_transformer.py

```diff
@@ -548,25 +551,26 @@ def forward(
             position_ids = torch.arange(past_length, input_shape[-1] + past_length, dtype=torch.long, device=device)
             position_ids = position_ids.unsqueeze(0)
 
-        # GPT2Attention mask.
+        # Attention mask.
```

Comment:
Here I would use # ignore copy - the model shouldn't have FA2 logic

Reply:
Addressed